tests/components/bosch_shc/test_config_flow.py

```diff
@@ -0,0 +1,130 @@
+"""Test the Bosch SHC config flow."""
+from boschshcpy import SHCSession
+
+from homeassistant import config_entries, setup
+from homeassistant.components.bosch_shc.config_flow import CannotConnect, InvalidAuth
+from homeassistant.components.bosch_shc.const import DOMAIN
+
+from tests.async_mock import patch
+
+
+async def test_form(hass):
+    """Test we get the form."""
+    await setup.async_setup_component(hass, "persistent_notification", {})
+    result = await hass.config_entries.flow.async_init(
+        DOMAIN, context={"source": config_entries.SOURCE_USER}
+    )
+    assert result["type"] == "form"
+    assert result["errors"] == {}
+
+    with patch(
+        "homeassistant.components.bosch_shc.config_flow.validate_input",
```

Comment:
Patch the library instead of our code in the config flow. That will make the coverage check pass.